fix docs for VM live migration network (#72586)
openshift/openshift-docs: modules/virt-configuring-secondary-network-vm-live-migration.adoc

    "cniVersion": "0.3.1",
    "name": "migration-bridge",
    "type": "macvlan",
    "master": "eth1", <3>
    "mode": "bridge",
    "ipam": {
      "type": "whereabouts", <4>
      "range": "10.200.5.0/24" <5>
    }
  }'
----
<1> Specify the name of the `NetworkAttachmentDefinition` object.

> <2> The `NetworkAttachmentDefinition` object must be created in the {CNVNamespace} namespace.

Reviewer: ```suggestion
<2>  Specify the namespace where the `NetworkAttachmentDefinition` object resides. This must be `{CNVNamespace}`.
```

<3> Specify the name of the NIC to be used for live migrations.

> <4> Specify the name of the CNI plugin that provides the network for the `NetworkAttachmentDefinition`.

Reviewer: ```suggestion
<4> Specify the name of the CNI plugin that provides the network for the network attachment definition.
```
